setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020  The Project U-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+import setuptools
+
+setuptools.setup(
+    name="prjuray",
+    version="0.0.1",
+    author="SymbiFlow Authors",
+    author_email="symbiflow@lists.librecores.org",
+    description="Project U-Ray database access code",
+    long_description="",
+    url="https://github.com/SymbiFlow/prjuray-tools",
+    packages=setuptools.find_packages(),
+    install_requires=['intervaltree', 'simplejson'],
```

Comment:
Can you explain why we put `intervaltree` and `simplejson` here, and not in requirements?  Should it be removed from both places?

Reply:
These two packages are required by the `prjuray` package. That is why they are specified in the setup script rather than in requirements.txt.

I see it this way: `setup.py` + everything under `prjuray/` is the Python package with database access code. The rest of the repository (including `requirements.txt`) is something that uses that package. That's why I added `-e ./` to `requirements.txt` and dependency on intervaltree + simplejson to `setup.py`.